tests/api-testing/tests/test_foldersnew.py

```diff
@@ -0,0 +1,142 @@
+def retrieve_new_dashboard(session, base_url, org_id, folder_id):
+    resp_get_dashboard = session.get(
+        f"{base_url}api/v2/{org_id}/folders/dashboards/{folder_id}")
+    print(resp_get_dashboard.content)
+    assert (
+        resp_get_dashboard.status_code == 200
+    ), f"Retrieve these dashboards: Expected 200, but got {resp_get_dashboard.status_code} {resp_get_dashboard.content}"
+
+def delete_new_dashboard(session, base_url, org_id, folder_id):
+    resp_delete_dashboard = session.delete(
+        f"{base_url}api/v2/{org_id}/folders/dashboards/{folder_id}")
+    assert (
+        resp_delete_dashboard.status_code == 200
+    ), f"Deleting these dashboards: Expected 200, but got {resp_delete_dashboard.status_code} {resp_delete_dashboard.content}"
```

Comment:
This test is deleting the "dashboard folder" not the dashboard itself. I'd recommend updating to update the logs to reflect that in order to avoid confusion.